README.md

```diff
@@ -259,6 +259,68 @@ A JSON structured response within the body containing the listing, or an iRODS e
 }
 ```
 
+### /logicalpath
+Deletes a data object or a collection.
+
+**Method** DELETE
+
+**Parameters**
+- logical-path: The absolute path leading to the data object or collection to be deleted.
+- no-trash: Don't send to trash, delete permanently. Optional, defaults to false.
+- recursive: Recursively delete contents of a collection. Optional, defaults to false.
+- unregister: Unregister data objects instead of deleting them. Optional, defaults to false.
+
+**Example CURL command**
+```
+curl -X DELETE -H "Authorization: ${TOKEN}" "http://localhost:80/irods-rest/0.9.1/logicalpath?logical-path=/tempZone/home/rods/hello.cpp&no-trash=1"
+```
+
+**Returns**
+Nothing on success.
+On failure, returns a JSON object with a description of the error. For example, the following CURL
+passes a non-existent path for deletion.
+
+```
+curl -X DELETE -H "Authorization: ${TOKEN}" "http://localhost:80/irods-rest/0.9.0/logicalpath?logical-path=blah!"
```

Comment:
0.9.0... we'll have to touch all these eventually, but let's at least get them consistent when adding new ones.